style.css

```diff
@@ -0,0 +1,88 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+#header-section {
+  width: 100%;
+  height: 104px;
+  background-image: url('images/logo-image.svg');
+  background-repeat: no-repeat;
+  background-position: top left;
+  background-size: contain;
+}
+
+#header-section div {
+  display: none;
+  float: left;
+  height: 74px;
+  width: 74px;
+}
+
+#header-section nav a {
+  float: right;
+}
+
+#section-main {
+  width: 100vw;
+  display: block;
+  padding: 0 12px;
+}
```

Comment:
- [ ] As I pointed out before, it is required to use flexbox to align the contents in this section. Use the link provided to read more on how to do this.

Reply:
- [ ] As the previous reviewer has emphasized, please use flexbox to align the contents of each section. Flexbox helps out a lot when implementing the desktop version using media queries. :)